src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py

```diff
@@ -6,8 +6,9 @@
 from collections import OrderedDict
 
 _property_map = {
-    'name': 'NAME',
-    'resourceGroup': 'RESOURCE GROUP',
+    #'name': 'NAME',
+    #'resourceGroup': 'RESOURCE GROUP',
```

Comment:
This will break other commands such as `az acr list -o table` where we do want to show resource group. Can you separate your format method out?

Reply:
Sure. I will create a separate format for the build and leave the format for the custom, crendential, repository, webhook and replication untouched.